src/prometheus.rs

```diff
@@ -0,0 +1,189 @@
+use std::{env, error::Error};
+
+use prometheus::register_gauge;
+use rustic_core::repofile::SnapshotFile;
+
+pub fn parse_label(
+    s: &str,
+) -> Result<(String, String), Box<dyn Error + Send + Sync + 'static>> {
```

Comment:
I wonder if we can use clap for parsing this, but if not, please use `anyhow::Result` here..

Reply:
The reason I did it this way is based on [this page of the clap documentation](https://docs.rs/clap/latest/clap/_cookbook/typed_derive/index.html), so I'd assume clap offers no built-in way to do this. I just adjusted to `anyhow::Result`! :)